<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Moonstone :: MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/static/default/icon/22x22/moonstone.png</normaloff>:/static/default/icon/22x22/moonstone.png</iconset>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QToolBar" name="toolBarActions">
   <property name="windowTitle">
    <string>Action Window</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpenProject"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionSplitLeftNew"/>
   <addaction name="actionSplitLeftClose"/>
   <addaction name="separator"/>
   <addaction name="actionShowPlaneLines"/>
  </widget>
  <widget class="QDockWidget" name="toolProperties">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>215</width>
     <height>119</height>
    </size>
   </property>
   <property name="windowIcon">
    <iconset resource="resources.qrc">
     <normaloff>:/static/default/icon/22x22/moonstone.png</normaloff>:/static/default/icon/22x22/moonstone.png</iconset>
   </property>
   <property name="windowTitle">
    <string>Tool Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBarTools_2">
   <property name="windowTitle">
    <string>Tools</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>612</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpenProject"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionFullScreen"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelpContents"/>
    <addaction name="actionReleaseNotes"/>
    <addaction name="actionReportBugs"/>
    <addaction name="separator"/>
    <addaction name="actionCheckForUpdates"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionLicense"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBarTools">
   <property name="windowTitle">
    <string>Tools</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/static/default/icon/48x48/document-new.png</normaloff>:/static/default/icon/48x48/document-new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Import DICOMs</string>
   </property>
   <property name="iconText">
    <string>Import DICOMs</string>
   </property>
   <property name="toolTip">
    <string>Import DICOMs</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/static/default/icon/22x22/moonstone-about.png</normaloff>:/static/default/icon/22x22/moonstone-about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/static/default/icon/22x22/help-about.png</normaloff>:/static/default/icon/22x22/help-about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;License</string>
   </property>
  </action>
  <action name="actionRecentProjects">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/static/default/icon/22x22/document-open-recent.png</normaloff>:/static/default/icon/22x22/document-open-recent.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Recent Projects</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/static/default/icon/48x48/document-save.png</normaloff>:/static/default/icon/48x48/document-save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/static/default/icon/22x22/document-save-as.png</normaloff>:/static/default/icon/22x22/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/static/default/icon/22x22/document-print.png</normaloff>:/static/default/icon/22x22/document-print.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/static/default/icon/22x22/window-close.png</normaloff>:/static/default/icon/22x22/window-close.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/static/default/icon/22x22/application-exit.png</normaloff>:/static/default/icon/22x22/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSaveAll">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/static/default/icon/22x22/document-save-all.png</normaloff>:/static/default/icon/22x22/document-save-all.png</iconset>
   </property>
   <property name="text">
    <string>Save A&amp;ll..</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/static/default/icon/48x48/project-open.png</normaloff>:/static/default/icon/48x48/project-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Import Project</string>
   </property>
   <property name="iconText">
    <string>Import Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/static/default/icon/22x22/configure.png</normaloff>:/static/default/icon/22x22/configure.png</iconset>
   </property>
   <property name="text">
    <string>Prefere&amp;nces</string>
   </property>
   <property name="iconText">
    <string>Preferences</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionHelpContents">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/static/default/icon/22x22/help-contents.png</normaloff>:/static/default/icon/22x22/help-contents.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Help Contents</string>
   </property>
  </action>
  <action name="actionReleaseNotes">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/static/default/icon/22x22/im-status-message-edit.png</normaloff>:/static/default/icon/22x22/im-status-message-edit.png</iconset>
   </property>
   <property name="text">
    <string>Release &amp;Notes</string>
   </property>
  </action>
  <action name="actionReportBugs">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/static/default/icon/22x22/tools-report-bug.png</normaloff>:/static/default/icon/22x22/tools-report-bug.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Report Bugs</string>
   </property>
  </action>
  <action name="actionCheckForUpdates">
   <property name="text">
    <string>Check f&amp;or Updates...</string>
   </property>
  </action>
  <action name="actionSplitLeftNew">
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/static/default/icon/48x48/view-right-new.png</normaloff>:/static/default/icon/48x48/view-right-new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Show side window</string>
   </property>
  </action>
  <action name="actionSplitLeftClose">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/static/default/icon/48x48/view-left-close.png</normaloff>:/static/default/icon/48x48/view-left-close.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Hide side window</string>
   </property>
  </action>
  <action name="actionFullScreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/static/default/icon/22x22/view-fullscreen.png</normaloff>:/static/default/icon/22x22/view-fullscreen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Fullscreen</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionShowPlaneLines">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/static/default/icon/48x48/office-chart-polar.png</normaloff>:/static/default/icon/48x48/office-chart-polar.png</iconset>
   </property>
   <property name="text">
    <string>Show Plane Lines</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+P</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
